wazo_acceptance/steps/user.py

```diff
@@ -414,3 +414,32 @@ def then_user_src_has_a_call_recording_with_user_dst(context, firstname_src, las
     assert cdr[0]['source_user_uuid'] == user_src['uuid']
     assert cdr[0]['destination_user_uuid'] == user_dst['uuid']
     assert len(cdr[0]['recordings']) == 1
+
+
+@then('"{firstname_src} {lastname_src}" has {count} call recordings with "{firstname_dst} {lastname_dst}"')
+def then_user_src_has_n_calls_recording_with_user_dst(context, firstname_src, lastname_src, count, firstname_dst, lastname_dst):
+    user_src = context.helpers.confd_user.get_by(firstname=firstname_src, lastname=lastname_src)
+    user_dst = context.helpers.confd_user.get_by(firstname=firstname_dst, lastname=lastname_dst)
+    cdr = context.call_logd_client.cdr.list(user_uuid=user_src['uuid'])['items']
+    assert cdr[0]['source_user_uuid'] == user_src['uuid']
+    assert cdr[0]['destination_user_uuid'] == user_dst['uuid']
+    assert len(cdr[0]['recordings']) == int(count)
+    files = set()
+    for recording in cdr[0]['recordings']:
+        media = context.call_logd_client.cdr.get_recording_media(cdr[0]['id'], recording['uuid'])
+        files.add(media)
+    assert len(files) == int(count)
```

Comment:
Why use a set for the files if we are asserting on the number of files?

Reply:
I'm checking for file that would get overwritten. If the same filename is used to record on the filesystem the API will return 2 recordings but the media will be the same for both, since both files will be generated from the same file on the filesystem.

Using a set will only count different files. If the file was overwritten, both files will have the same data and the length of the set will be less than the number of recordings.

Reply:
Can you add a comment beside the `set()` to explain it (or in the commit)
Because the next time someone will refactor/edit this function, it will be very hard to know this behavior

Reply:
I added a message to the assertion and renamed my set to `unique_media`. Does that make sense?

Reply:
yes thank you :+1: